src/sentry/incidents/models.py

```diff
@@ -401,6 +405,21 @@ def created_by(self):
             pass
         return None
 
+    def owner_id(self):
+        if self.user:
+            return f"user:{self.user_id}"
+
+        if self.team:
+            return f"team:{self.team_id}"
+
+    def owner(self):
```

Comment:
I think this makes more sense as a `@property`

Reply:
Me too 😄